app/initializers/model-fragments.js

```diff
@@ -1,4 +1,6 @@
-import _ from 'model-fragments';
+// Import the fragment base class to ensure monkey-patching side effects are
+// applied before any store instances are created :(
+import _ from 'model-fragments/fragment';
```

Comment:
Haha derp, I knew that I swear :wink: 

I actually realized that I need to revert the `/fragments` portion, because the index file is where MF gets registered with `Ember.libraries`.